tox.ini

```diff
@@ -1,29 +1,73 @@
+# Generated from:
+# https://github.com/zopefoundation/meta/tree/master/config/pure-python
 [tox]
 envlist =
-   flake8,py27,py35,py36,py37,py38,pypy,pypy3,coverage,docs
+    lint
+    py27
+    py35
+    py36
+    py37
+    py38
+    py39
+    pypy
+    pypy3
+    docs
+    coverage
 
 [testenv]
+usedevelop = true
+deps =
+    zope.testrunner
 commands =
     zope-testrunner --test-path=src []
-extras = test
+extras =
+    test
 
-[testenv:coverage]
-usedevelop = true
-basepython =
-    python3.7
-commands =
-    coverage run -m zope.testrunner --test-path=src []
-    coverage report --fail-under=100
```

Comment:
We had 100%, and now we have 99%?  What happened?

Reply:
It's mostly branch coverage (which we had disabled before), but also now line 109 in src/zope/sendmail/maildir.py is no longer covered.

https://github.com/zopefoundation/zope.sendmail/blob/5e3f7868607f9e52f7d029c152b9bc18ce4b6474/src/zope/sendmail/maildir.py#L109

This appears to be a side effect of switching from basepython = python3.7 to basepython = python3 (which is 3.8 on my Ubuntu 20.10)!

Anyway, this is fine, we can climb back to 100% later.